tests/plugin.test.ts

```diff
@@ -144,6 +145,49 @@ describe("Purchases", () => {
     );
   });
 
+  describe("setLogLevel", () => {
+    // If the enum is backed by string values, iterating over the enum won’t work because the `value` in our loop is a
+    // string instead of a number. The indexer of `LOG_LEVEL` can only accept `VERBOSE`, `DEBUG`...
+    // We need a helper method to create a new type `enumKeys`
+    for (const value of enumKeys(LOG_LEVEL)) {
```

Comment:
Should we also have a test checking `enumKeys(LOG_LEVEL)` has the values we want? To make sure we don't miss any